SwrveSDK/src/main/java/com/swrve/sdk/SwrveAssetsManager.java

```diff
@@ -0,0 +1,69 @@
+package com.swrve.sdk;
+
+import java.io.File;
+import java.util.HashSet;
+import java.util.Set;
+
+public class SwrveAssetsManager {
+    private final String LOG_TAG = "SwrveAssetsManager";
```

Comment:
The main reason I've been using different tags (usually with the SwrvePrefix or name of the class) is because it makes it easier to track down where a log is coming from. 

If they all have SwrveSDK as the tag then we have to search by the log message.